ProcessDoctor.Backend.Windows/NT/RTL_USER_PROCESS_PARAMETERS.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.InteropServices;
+using PInvoke;
+
+namespace ProcessDoctor.Backend.Windows.NT;
+
+[StructLayout(LayoutKind.Sequential)]
+public unsafe struct RTL_USER_PROCESS_PARAMETERS
+{
+    private fixed byte Reserved1[16];
+
+    private fixed int Reserved2[20];
```

Comment:
Honestly, I don't know how to write a fixed `PVOID` array any other way. Maybe you have some ideas?

Reply:
Replaced `int` with `long` and returned the standard array size. It seems to be working 😁